lib/mongoid/association/options.rb

```diff
@@ -112,8 +112,10 @@ def touch_field
         @touch_field ||= options[:touch] if (options[:touch].is_a?(String) || options[:touch].is_a?(Symbol))
       end
 
-      private
-
+      # Whether the association object should be automatically touched
+      # when its inverse object is updated.
+      #
+      # @return [ true, false ] Default is false.
```

Comment:
# @return [ true | false ] returns true if this association is
#   automatically touched, false otherwise. The default is false.

Reply:
Done